### ISSUE TITLE
Missing empty keyrow but the xml contains

### ISSUE THREAD

**Issue by Meister1977** [AnySoftKeyboard/AnySoftKeyboard#3778]
### Steps to reproduce
1. Install latest beta
2. Install Hungarian keyboard
3.https://github.com/Meister1977/anysoftkeyboard-hungarian/blob/master/src/main/res/xml/hu_qwerty_swype.xml

### Expected behaviour
![Screenshot_20230731-230506](https://github.com/AnySoftKeyboard/AnySoftKeyboard/assets/10465690/0ee402f8-f5b0-43c6-a9a5-722f02e238ad)

There should be an empty row between the clipboard button and the qwertz line.

### Actual behaviour
There is no empty row, so I press the clipboard buttons randomly.

**Android OS version:**
13
**Device manufacturer and model:**
Pixel 7 Pro
**List of installed add-ons (like languages, or themes):**
-

**Comment by GoRaN909** [comment]
May I ask you is that the original unmodified version of AnySoftKeyboard or it's your speciall one becuase it look your Top-Row is different from ours! How did you placed the Suggestions bar under the top row(Toolbar) ? or How did you make clipboard button on the left side? @Meister1977

**Comment by Meister1977** [comment]
> May I ask you is that the original unmodified version of AnySoftKeyboard or it's your speciall one becuase it look your Top-Row is different from ours! How did you placed the Suggestions bar under the top row(Toolbar) ? or How did you make clipboard button on the left side? @Meister1977

This is the Android Play version (beta) of the ASK.
The suggestion bar is the upper bar.
Bellow the suggestion bar there is the 1234!#() row, and bellow that therebis the first keyboard row specified in the layout xml.
It contains Kivág (cut), Másol (copy), Beilleszt (paste) and Mind kijelöl (select all).
The problem: it should be some spaces between this row and the qwertz row to avoid accidental select all and cut.
The layout xml contains empty rows. In the non-beta ask there is some spaces.

**Comment by GoRaN909** [comment]
No no you misunderstanding me I meant how did you move the suggestions bar itno here i cropped this bar that on the red arrows 
👇
![257350504-0ee402f8-f5b0-43c6-a9a5-722f02e238ad](https://github.com/AnySoftKeyboard/AnySoftKeyboard/assets/3160786/3ba60aa1-1e8c-405f-be15-7cb8bf6ae2a7)

**Comment by Meister1977** [comment]
> No no you misunderstanding me I meant how did you move the suggestions bar itno here i cropped this bar that on the red arrows 👇 ![257350504-0ee402f8-f5b0-43c6-a9a5-722f02e238ad](https://user-images.githubusercontent.com/3160786/260816408-3ba60aa1-1e8c-405f-be15-7cb8bf6ae2a7.png)

This is not the suggestion bar.
See this, you can see, the suggestion bar is above the keyboard.

![Screenshot_20230815-234854](https://github.com/AnySoftKeyboard/AnySoftKeyboard/assets/10465690/46d191a9-5d64-417e-9e6c-339dbe18e1bc)

**Comment by GoRaN909** [comment]
Oh my fault that's a part of your own layout you added the text-editor on the first row.

`    <Row android:keyHeight="@integer/key_short_height">
        <Key android:keyWidth="8%p" android:codes="-100"  android:keyIcon="@drawable/menu" android:keyEdgeFlags="left"/>
        <Key android:keyWidth="21%p" android:codes="@integer/key_code_clipboard_cut" android:keyLabel="@string/hungarian_keyboard_cut"/>
        <Key android:keyWidth="21%p" android:codes="@integer/key_code_clipboard_copy" android:keyLabel="@string/hungarian_keyboard_copy" />
        <Key android:keyWidth="21%p" android:codes="@integer/key_code_clipboard_paste" android:keyLabel="@string/hungarian_keyboard_paste"/>
        <Key android:keyWidth="21%p" android:codes="@integer/key_code_clipboard_select_all" android:keyLabel="@string/hungarian_keyboard_selectall" />
        <Key android:keyWidth="8%p" android:codes="@integer/key_code_cancel" android:keyIcon="@drawable/close_suggestions_strip_icon" android:keyEdgeFlags="right"/>`

**Comment by Meister1977** [comment]
> Oh my fault that's a part of your own layout you added the text-editor on the first row.
> 
Yes, and in my xml I tried to add empty row bellow this buttons, because if you type fast, and misstype a Select All-->Cut, then your all text goes away. Without UNDO. You can start re-type all again.
So it would be really important to add the empty line, as it is in the non-beta app. (Or if there is other way to add padding bottom, it is OK for me too. But in this case it must work in LIVE and BETA version too, to keep the same layout.

**Comment by GoRaN909** [comment]
> Yes, and in my xml I tried to add empty row bellow this buttons, because if you type fast, and misstype a Select All-->Cut, then your all text goes away. Without UNDO. You can start re-type all again. So it would be really important to add the empty line, as it is in the non-beta app. (Or if there is other way to add padding bottom, it is OK for me too. But in this case it must work in LIVE and BETA version too, to keep the same layout.

Now I understood what you meant you want more space between main buttons and text editor tool I don't think this keyboard support key spaces why you not try this one [Florisboard](https://github.com/florisboard/florisboard) it have too many good features and customization also it have an option to change key spacing between buttons.

**Comment by Meister1977** [comment]
> Now I understood what you meant you want more space between main buttons and text editor tool I don't think this keyboard support key spaces


In the non-beta version I could add the space as an empty row. So it is a regression. A bug.
